solutions/chapter_11/test/specs/mobile.app.spec.ts

```diff
@@ -0,0 +1,32 @@
+import {$, browser, expect} from '@wdio/globals'
+
+describe('Mobile Application', () => {
+  beforeEach(async () => {
+    await relaunchApp()
+    await $('~Home-screen').waitForDisplayed()
+    await $('~Login').click()
+    await $('~button-LOGIN').waitForDisplayed()
+  })
+
+  it('should be able to create an element snapshot', async () => {
+    await expect($('~button-LOGIN')).toMatchElementSnapshot('login-button', {hideElements: [await $('nav.navbar')]})
+  })
+
+  it('should be able to create a device snapshot', async () => {
+    await expect(browser).toMatchScreenSnapshot('app-forms', {blockOutToolBar: true, blockOutStatusBar: true})
+  })
+})
+
+/**
+ * Simple function to relaunch the app
+ */
+async function relaunchApp() {
```

Comment:
This might be nice to add as a WebdriverIO command. I've seen a lot of people asking for this since Appium as deprecated the protocol command.

Reply:
Created this FR https://github.com/webdriverio/webdriverio/issues/12626